src/detection.c

```diff
@@ -96,3 +98,18 @@ void get_gyro_raw(uint16_t *vo, uint16_t *vref)
 	*vo = adc_read_injected(ADC2, 1);
 	*vref = adc_read_injected(ADC2, 2);
 }
+
+/**
+ * @brief Function to get data from sensors.
+ *
+ * - Return the values of sensors with emitter on and off.
```

Comment:
I'd rather use `@return` here. :wink:

Reply:
Instead of @return, I mix the two sentences into a one more descriptive.